app/components/developer_query_component.rb

```diff
@@ -18,6 +18,15 @@ def time_zones
       .map { |offset| [offset, "#{offset} #{t("developer_query_component.gmt")}"] }
   end
 
+  def role_types
+    %i[part_time_contract full_time_contract full_time_employment]
```

Comment:
This is now the third or fourth time the codebase is iterating over these keys. I'm worried if we update these we won't catch everywhere we use them.

Can you move this to a constant on `RoleType` and refactor the others?

```ruby
class RoleType < ApplicationRecord
  TYPES = %i[part_time_contract full_time_contract full_time_employment].freeze

  # ...
end

puts RoleType::TYPES
```

Reply:
Sure that makes sense.